lightblue/src/main/java/org/esbtools/eventhandler/lightblue/client/BulkLightblueRequester.java

```diff
@@ -85,7 +85,7 @@ public BulkLightblueRequester(LightblueClient lightblue) {
     }
 
     @Override
-    public TransformableFuture<LightblueDataResponses> request(AbstractLightblueDataRequest... requests) {
+    public TransformableFuture<LightblueDataResponses> request(CRUDRequest... requests) {
```

Comment:
You cannot use DataBulkRequest with anything other than a CRUDRequest, and bulk operations appear to be what the EventHandler users. 

That said, the below interfaces could each be altered to take a generic that extends LightblueDataRequest. Then BulkLightblueRequester could just use CRUDRequest. We could go an extra step and create interface helpers for CRUDRequest. See example below.

Honestly, it seems a bit much to me as BulkLightblueRequester is currently the only implementation of LightblueRequester.

- LightblueDataResponses
- LightblueRequester
- LightblueResponses

Example
```
public interface LightblueRequester<T extends LightblueDataRequest> extends Requester<T, LightblueDataResponse> { ... }

public interface LightblueCrudRequester extends LightblueRequester<CRUDRequest> {}
```